### ISSUE TITLE
Port Forwarding for WSL not working (anymore)

### ISSUE THREAD

**Issue by 9cento** [oobabooga/text-generation-webui#474]
TL;DR: I did the procedure as instructed under #380, it worked, I un-did it because stuff, I did it again (and again, double-checking it) but it doesn't work anymore no matter what (can't connect from another device on the same private network). What's happening?

So I did everything posted in #380 and managed to get the thing to work as intended, I also put in place a script in WSL Ubuntu to revert its ip to the one specified in `netsh interface portproxy add...` (so pseudo-static) and the setup was even surviving reboots. Then for an unrelated issue (WSL Ubuntu unable to connect to the internet anymore, will investigate later) I did `netsh interface portproxy reset`, proceeded to do my stuff with the re-enabled internet connection in WSL and then re-did the procedure from scratch again and again (including `netsh interface portproxy reset`), to my surprise the procedure didn't work anymore. How comes?

**Comment by naelonambul** [comment]
I think  you might need this.
I run this powershell script when I boot my pc.
https://github.com/microsoft/WSL/issues/4150#issuecomment-504209723

**Comment by 9cento** [comment]
> I think you might need this. I run this powershell script when I boot my pc. [microsoft/WSL#4150 (comment)](https://github.com/microsoft/WSL/issues/4150#issuecomment-504209723)

That's a solid alternative, thanks for sharing. Will try asap, just to be clear: what should I undo (other than my older script) before running this script you shared? Basically I set port forwarding through `netsh interface portproxy` and added a firewall rule for port 7860, will deleting the rule and running `netsh interface portproxy reset` suffice?

**Comment by naelonambul** [comment]
just remove before what you did like 7860 port.
if you want more clear method. do reset.
and then do script after you reboot.

**Comment by 9cento** [comment]
> just remove what you did like 7860 port. if you want more clear method. do reset. and then do script after you reboot.

Yep, working like a charm. Thanks again
